src/site/content/en/lighthouse-performance/uses-long-cache-ttl/index.md

```diff
@@ -51,8 +51,11 @@ Cache-Control: max-age=31536000
 ```
 
 The `max-age` directive tells the browser how long it should cache the resource, in seconds.
-`31536000` corresponds to 1 year: 60 seconds * 60 minutes * 24 hours * 365 days =
-31536000 seconds.
+`31536000` corresponds to 1 year:
+
+```text
```

Comment:
Why put this in a code block?

Reply:
Initially I was worried that the `*` characters would cause the text to italicize. But then I realized that it just looks nicer this way. The text immediately preceding this also puts its example in a code block so it seemed more consistent?

Reply:
I guess my concern is that this isn't actually code--it's just explaining the code. Here's the GDDSG page about what gets put in code font: https://developers.google.com/style/code-in-text

But go with whatever you think makes sense!

Reply:
I removed it from the code block and swapped the asterisk for the utf-8 multiplication symbol ×. Bonus points is it now announces correctly in a screen reader.